app-modules/assistant/database/migrations/2023_09_11_214023_create_assistant_chat_messages_table.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('assistant_chat_messages', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->foreignUuid('assistant_chat_id')->constrained('assistant_chats');
+            $table->longText('message');
+            $table->enum('from', ['user', 'assistant']);
```

Comment:
I believe this is different for Postgres. Postgres has much more advanced support for the enum column. It can be changed without deleting the column, etc.

I put this here to propose using it to add some data integrity to the DB side in some places.

I am also fine using a string-backed enum as well or completely instead.

What do you think?

Reply:
Ah you know what, nvm. I did some more research and apparently you can't delete enum values once created. So I will change this to a string.

Reply:
Changed